### ISSUE TITLE
why using low learning rate?

### ISSUE THREAD

**Issue by hunterlew** [HolmesShuan/AlexNet-BN-Caffemodel-on-ImageNet#2]
In my opinion, BN layer allows higher initial lr. Have u evaluated the performance with an initial lr 0.1?

**Comment by HolmesShuan** [comment]
`solver.prototxt` is just a guideline for fine-tuning. In our evaluation, we started from `lr=0.05`.

**Comment by hunterlew** [comment]
@HolmesShuan Well, still notice that you used dropout with bn layers. Have u evaluated the performance using bn without dropout? My experiment proves that bn layers do accelerate the training, but only got 53.5% top-1 error rate. Hope for your true solver.prototxt and train.prototxt!

**Comment by HolmesShuan** [comment]
`dropout` is required. solver.prototxt has been updated. During the training process, we fine-tuned the learning rate and step size several times. To be honest, we can not recover the original solver.prototxt. If you are still willing to know more details about BN-AlexNet, [DIGITS](https://github.com/NVIDIA/DIGITS) may suit you well.
